samples/bluetooth/mesh/light_dimmer/src/model_handler.c

```diff
@@ -140,6 +151,42 @@ static void dimmer_button_handler(struct dimmer_ctx *ctx, bool pressed)
 	}
 }
 
+static struct scene_btn_ctx scene_btn = {
+	.current_scene = 1,
+	.pressed_time = 0
+};
+
+static void scene_button_handler(struct scene_btn_ctx *ctx, bool pressed)
+{
+	if (pressed) {
+		ctx->pressed_time = k_uptime_get();
+	} else {
+		int err;
+
+		if (k_uptime_get() - ctx->pressed_time > SHORT_PRESS_THRESHOLD_MS) {
+			/* Long press, store scene */
+			LOG_INF("Storing scene %d", ctx->current_scene);
```

Comment:
Just a suggestion to improve user-friendliness: I would appreciate a LED light signal on the controller device indicating that the user has been pressing the button for long enough for the scene to be stored on the network nodes. The light dimming operation has visual feedback but the scene store op has not. 

If we choose to do this, remember to update documentation as well :)